### PR TITLE
apicurio installation

### PR DESCRIPTION
**Pull request by odra**
## Motivation

https://trello.com/c/N42Qvvkk/133-8-installer-add-apicurio-operator-for-installation

## What

Adds apicurio operator installation

## Why

To enable apicurio usage in an integreatly instance.

## Verification Steps

Run the installer:

```
ansible-playbook -i inventories/hosts --skip-tags='gitea' playbooks/install.yml
```

Apicurio should be available afterwards.

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

File: evals/roles/apicurio/tasks/keycloak-client.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    url: "https://{{ apicurio_kc_host }}/auth/realms/master/protocol/openid-connect/token"
+    method: POST
+    body: "client_id=admin-cli&username={{ apicurio_kc_username }}&password={{ apicurio_kc_password }}&grant_type=password"
+    validate_certs: "{{ apicurio_validate_certs }}"
+  register: apicurio_kc__auth_response
+  retries: 10
+  delay: 2
+  until: apicurio_kc__auth_response.status == 200
+
+- name: Create public client in {{ apicurio_kc_realm }} realm
```
Comment (maleck13) [inline evals/roles/apicurio/tasks/keycloak-client.yml]: are we not adding this to the rhsso crd? ping @philbrookes

**Comment by odra** [comment]
other components are still using the same scheme but I am fine in waiting for the keycloak change.

I was also wondering if we needed a postgresql operator as well as most components require one.

**Comment by maleck13** [comment]
@odra doing a quick verification on pds

**Comment by maleck13** [comment]
retrying

**Comment by maleck13** [comment]
looks like it needs a rebase

**Pull request closed without merging**